site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -62,14 +62,14 @@ export const UsersPage: FC<{ children?: ReactNode }> = () => {
     paginationRef,
     count,
   } = usersState.context;
+  const permissions = usePermissions();
```

Comment:
Does the return value of `usePermissions` change each time you call it? The arguments don't change between the call on line 65 vs the call on line 67, so I would expect the properties to be the same, at least while inside the same render.

Reply:
They don't change, but I'm not sure what is your concern here 🤔

Reply:
I guess I'm just wondering if this couldn't be consolidated to one `usePermissions` call, if both calls expose the same value for a given render.

Reply:
Not sure if I get it, where do you see two `usePermissions` call?

Reply:
Whaaat? Let me fix this.